prover/build

```diff
@@ -100,7 +103,7 @@ def known_passing(test):
 def make_test(rule, test):
     commandline = "'--default-strategy %s'" % strategy_for_test(test)
     return proj.source(test) \
-               .then(rule.variable('flags', '-cCOMMANDLINE=%s' % commandline))
+               .then(rule.variable('flags', ('-cCOMMANDLINE=%s' % commandline) + ' ' + proverPathFlag()))
```

Comment:
Lets not mix `%` based formatting with string concatenation. Can we add this flag to the `CommandLine` grammar instead of adding a new parameter. That way we would get better error messages when this required flag is not passed.

Reply:
We can; just give me an example of how should that parameter look when we call `krun`.

Reply:
HMm, actually this should be changed to use the `./prover` script instead. I'll add a commit that does that later today